fenix/app/src/main/res/raw/keep.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools"
+    tools:keep="@drawable/onboarding_default_browser*,@drawable/onboarding_sync*,@drawable/onboarding_notification*" />
```

Comment:
Let's name the files completely here so there's no wildcard in case there are other files with similar names in the future.